libs/designsystem/item/src/item.component.scss

```
@@ -41,8 +41,7 @@
     }
 
     @include utils.slotted('[slot="end"]') {
-      margin-inline-start: 0;
-      margin-inline-end: 0;
+      margin-inline: 0 0;
```

Consider simplifying further (same value):
```suggestion
      margin-inline: 0;
```